### PR TITLE
Fixing bug where it does not load on default page

### PR DESCRIPTION
**Pull request by Durbatuluk1701**
This seems to fix the bug

### CODE REVIEW

File: src/src/components/Contexts/CurrentViewContext.tsx [hunk 1]
```diff
@@ -65,6 +65,15 @@ const commonPrefix = `/calendar/u/0/r/`;
  * Extract current date from the Google Calendar URL
  */
 function parsePath(path: string): CurrentView | undefined {
+  if (path === '/calendar/u/0/r') {
+    // There is possibly more nuance to this, but I am not 100% sure. I think a sufficient fix for now
+    const today = new Date();
+    return {
+      view: 'day',
```
Comment (maxxxxxdlp) [inline src/src/components/Contexts/CurrentViewContext.tsx:72]: Uhgg... I have just discovered something bad.

The `/calendar/u/0/r` URL does not correspond to the day view, it uses the view that the user used last.
Thus, you will have to detect what view was rendered by Google Calendar in some way (either from the current value of the View switcher button or from the content)

Tell me if you would need help with that

Comment (maxxxxxdlp) [reply src/src/components/Contexts/CurrentViewContext.tsx:72]: Or if we need a short-term solution, treating `/calendar/u/0/r` as a week view might be good enough (as week view is the default first time you open google calendar)

**Review by maxxxxxdlp — CHANGES_REQUESTED** [review]

File: src/src/components/Contexts/CurrentViewContext.tsx (hunk 1 above)
Comment (Durbatuluk1701) [reply src/src/components/Contexts/CurrentViewContext.tsx:72]: I see. I will look into getting the view type, and if I can't get that then maybe we settle for the week view

Comment (Durbatuluk1701) [reply src/src/components/Contexts/CurrentViewContext.tsx]: This should be resolved by the new push

**Review by maxxxxxdlp — APPROVED** [review]

**Pull request merged**